web/modules/webspark/webspark_blocks/webspark_blocks.install

```diff
@@ -215,6 +215,13 @@ function webspark_blocks_update_9022(&$sandbox) {
   _webspark_blocks_revert_module_config();
 }
 
+/**
+ * WS2-1533 - Changed description text for field_media in Card, Card Degree, Card Story.
+ */
+function webspark_blocks_update_9025(&$sandbox) {
+  _webspark_blocks_revert_module_config();
+}
+
```

Comment:
@juanmitriatti Since there are A LOT of config files in this module, and we know exactly which files are updated for this fix, can we load in just the updated files? That will help avoid issues with updates timing out. Also, there was a merge conflict reported with this file, so maybe you can address that with this update, too.